lisa/base_tools/wget.py

```diff
@@ -90,3 +93,48 @@ def verify_internet_access(self) -> bool:
                 f"Internet is not accessible, exception occurred with wget {e}"
             )
         return False
+
+    @classmethod
+    def _windows_tool(cls) -> Optional[Type[Tool]]:
+        return WindowsWget
+
+
+class WindowsWget(Wget):
+    @property
+    def command(self) -> str:
+        return ""
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def get(
+        self,
+        url: str,
+        file_path: str = "",
+        filename: str = "",
+        overwrite: bool = True,
+        executable: bool = False,
+        sudo: bool = False,
+        force_run: bool = False,
+    ) -> str:
+        ls = self.node.tools[Ls]
+
+        fullpath = f"{file_path}\\{filename}"
+
+        # return if file exists and not overwrite
+        if ls.path_exists(file_path, sudo=sudo) and not overwrite:
+            self._log.debug(
+                f"File {fullpath} already exists and rewrite is set to False"
+            )
+
+        # TODO: add support for executables
+        # remove existing file if present and download
+        self.node.tools[Rm].remove_file(fullpath, sudo=sudo)
+        self.node.tools[PowerShell].run_cmdlet(
+            f"$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri '{url}'"
```

Comment:
Is the `"$ProgressPreference = 'SilentlyContinue` the same as "ErrorAction" parameter? If so, use ErrorAction is clear.

Reply:
This is different from error action. Error action suppress errors, but this command suppress display of download progress.